src/js/app.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-unused-vars */
```

Comment:
@thejoshsmith One minor thing - we probably don't need the eslint hints here any more as no longer assigning the import to a variable.

Reply:
Very good point @judereid! Have removed.